test/functional/interface_rest.py

```diff
@@ -211,7 +211,7 @@ def run_test(self):
 
         self.generate(self.nodes[0], 1)  # generate block to not affect upcoming tests
 
-        self.log.info("Test the /block, /blockhashbyheight, /headers, and /blockfilterheaders URIs")
+        self.log.info("Test the /block, /blockhashbyheight and /headers URIs")
```

Comment:
8af3ee098e9a3713b2e7851d875df885e6ab3548: should drop 25045 instead of reverting it